### PR TITLE
refactor(hack): add link to next hackathon

### PR DESCRIPTION
**Pull request by rofe**
Incorporated feedback from https://github.com/adobe/helix-cli/commit/7dfe0fc2fcab5411a7c670c39f71416429cf7bdc.

### CODE REVIEW

File: src/hack.cmd.js [hunk 1]
```diff
@@ -11,24 +11,40 @@
...
+    this._open = !!o;
     return this;
   }
 
   async run() {
     await this.init();
-    const url = `https://github.com/adobe/helix-home/tree/master/hackathons/${this._hackathon || ''}`;
-    // eslint-disable-next-line no-console
-    console.log(chalk`Check out the Helix Hackathon at {blue ${url}}`);
+    const url = `https://helix-home-adobe.project-helix.page/hackathons/${this._hackathon || ''}`;
```
Comment (trieloff) [inline src/hack.cmd.js]: https://helix-home-adobe.project-helix.page/hackathons/ is giving me an error 400, so `https://helix-home-adobe.project-helix.page/hackathons/README.html` should be the fallback.

Comment (rofe) [reply src/hack.cmd.js]: good catch!

**Comment by rofe** [comment]
I have no clue why codecov thinks the overall coverage drops by 30% :) there is only 1 loc which isn't covered by the test...

**Comment by trieloff** [comment]
@rofe as the tests are running in parallel, so does the code coverage gathering. Once the build is complete, you should see the accurate coverage.

**Pull request merged**